Sync/TabsPayload.swift

```diff
@@ -8,6 +8,10 @@ import Storage
 import XCGLogger
 import SwiftyJSON
 
+// Int64.max / 1000.
+private let MaxSecondsToConvertInt64: Int64 = 9223372036854775;
+private let MaxSecondsToConvertDouble: Double = Double(9223372036854775);
```

Comment:
nit: Can't we just set these equal to `Int64.max / 1000` and `Double(Int64.max / 1000)` respectively?